src/components/Form.js

```diff
@@ -98,34 +130,28 @@ export default function Form({
         onSubmit={handleSubmit(onSubmit)}
       >
         <Wrapper>
-          <LabelHeader htmlFor="destination">Destination:</LabelHeader>
-          <Counter name="counter of max characters for detination">
-            {counter}
-          </Counter>
-          <InputField
-            autoFocus
-            id="destination"
-            type="text"
-            placeholder="e.g. Lissabon..."
-            maxLength={40}
-            {...register('destination', {
-              onChange: e => {
-                setCounter(40 - e.target.value.length);
-              },
-              required: {
-                value: true,
-                message: 'The name of your next destination must be filled!',
-              },
-              minLength: 1,
-              maxLength: {
-                value: 39,
-                message:
-                  'You reached the max amount of allowed characters, try to keep it a littler shorter',
-              },
-            })}
-          />
-          {errors.destination?.message ? (
-            <ErrorMessage>{errors.destination?.message}</ErrorMessage>
+          <LabelHeader htmlFor="geocoderdestination">Destination:</LabelHeader>
+          {destinationMapbox === '' ? (
+            ''
+          ) : (
+            <InputField
+              id="geocoderdestination"
+              readOnly
+              disabled
+              value={preloadedValues.destination}
+            />
+          )}
+
+          <GeoCoderDestination
+            display={destinationMapbox !== '' ? 'none' : 'displayed'}
+            id={'geocoderdestination'}
```

Comment:
Is this an input? If so, then the label will be for both `GeoCoderDestination` and the conditionally rendered `InputField`, and that is breaking the label-input connection. You'd need separate labels.
```suggestion
            id="geocoderdestination"
```

Reply:
it's actually a `div` and inside the `div` as an `input` plus `label`
The id is for the mapbox api to identify where I want to set the search bar in my html